th2_data_services/data_source.py

```diff
@@ -254,7 +274,7 @@ def find_messages_by_id_from_data_provider(self, messages_id: Union[Iterable, st
                 logger.exception(exception_msg)
                 raise ValueError(exception_msg)
 
-            answer = change_pipeline_message(answer)
```

Comment:
Will we similarly give the option of receiving a message with and without adapters?